circle.yml

```diff
@@ -0,0 +1,67 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
+# DOCKER_USER
+# DOCKER_PASS
+#
+machine:
+  services:
+    - docker
+
+dependencies:
+  cache_directories:
+    - "~/docker"
+
+  pre:
+    - sudo apt-get update; sudo apt-get install pigz
+
+  override:
+    - docker info
+
+    # Build the container, using Circle's Docker cache. Only use 1 image per
+    # day to keep the cache size down.
+    - I="image-$(date +%j).gz"; if [[ -e ~/docker/$I ]]; then echo "Loading $I"; pigz -d -c ~/docker/$I | docker load; fi
+
+    # create a version.json
+    - >
+        printf '{"name":"%s","commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n'
+        "buildhub"
+        "$CIRCLE_SHA1"
+        "$CIRCLE_TAG"
+        "mozilla-services"
+        "$CIRCLE_PROJECT_REPONAME"
+        "$CIRCLE_BUILD_URL"
+        > version.json
+
+    # build the actual deployment container
+    - docker build --pull -t app .
+
+    # Clean up any old images; save the new one.
+    - I="image-$(date +%j).gz"; mkdir -p ~/docker; rm ~/docker/*; docker save app | pigz --fast -c > ~/docker/$I; ls -l ~/docker
+
+test:
+  override:
+    -  "true"
```

Comment:
Do we have buildhub tests that we could run here to make sure the image is working?

Reply:
> I didn't setup the unit tests task in the container because I was running into permissions issues (the app user is read-only right?)

I ran into the same issue for PollBot but I was able to write in /tmp